config.php

```diff
@@ -1,12 +1,22 @@
 <?php
-define('LANG', 'en');
+
+// Only set language if it is not already set in the querystring
+if(getenv('lang')) {
+    define('LANG', getenv('lang'));
+} else {
+    define('LANG', 'nl');
+}
```

Comment:
This might be a simpler notation :)

```suggestion
define('LANG', getenv('lang') ?: 'nl');
```